app/src/main/contrib/components/Responsibilities/Coverage/DownloadCoveragePage.tsx

```diff
@@ -31,7 +31,8 @@ export class DownloadCoveragePage extends ContribPageWithHeader<LocationProps> {
     }
 
     name() {
-        return "Download coverage data";
+        const s = responsibilityStore.getState();
+        return s.currentResponsibility.scenario.id;
```

Comment:
There's no way of distinguishing this from `UploadBurdenEstimatesPage` at the moment. Maybe use "Download coverage for [SCENARIO]".